app/src/pages/header/components/AddMovieForm.jsx

```diff
@@ -18,25 +20,40 @@ const ColoredButtonWrapper = styled(ColoredButton)`
 `
 
 function AddMovieForm(props) {
-    const [genres, setGenres] = useState([]);
-    
-    useEffect(() => {
-        Promise
-            .resolve(genresFromJson)
-            .then(genres => setGenres(genres))
-    }, [])
+    const [title, setTitle] = useState('');
+    const [releaseDate, setReleaseDate] = useState('');
+    const [url, setUrl] = useState('');
+    const [genres, setGenres] = useState('');

```

Comment:
all these stuff could be in the redux store partly - will talk today

Reply:
Will be done in the next [HW#7](https://github.com/aldaniluk/react-mentoring/pull/7)